src/current/v23.1/molt-verify.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Use the MOLT Verify tool
+summary: Learn how to use the MOLT Verify tool to check for post-migration data discrepancies.
+toc: true
+docs_area: migrate
+---
+
+The MOLT Verify tool checks for post-migration data discrepancies between a source and a target database.
+
+The tool performs the following verifications to ensure post-migration data integrity:
+
+- **Table Verification:** Check that the structure of tables between the source database and the target database are the same.
+- **Column Definition Verification:** Check that the column names, data types, constraints, nullability, and other attributes between the source database and the target database are the same. 
+- **Row Value Verification:** Check that the actual data in the tables are the same between the source database and the target database.
+
+## Supported databases
+
+The following databases are currently supported:
+
+- PostgreSQL
+- MySQL
+
+## Use the MOLT Verify tool
+
+1. Get the JDBC connection strings for the source and target databases you want to compare.
+2. Make sure the SQL user running the MOLT verify tool has access to the necessary tables.
```

Comment:
```suggestion
2. Make sure the SQL user running MOLT Verify has access to the necessary tables.
```

Reply:
does access mean read privileges? suggest linking to `grant.html#grant-privileges-on-specific-tables-in-a-database`

Reply:
I think they mean access to the Postgres/MySQL databases